src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1005,8 +1009,7 @@ def remove_all_hooks(self):
         """
         for _, model in self.components.items():
             if isinstance(model, torch.nn.Module) and hasattr(model, "_hf_hook"):
-                is_sequential_cpu_offload = isinstance(getattr(model, "_hf_hook"), accelerate.hooks.AlignDevicesHook)
-                accelerate.hooks.remove_hook_from_module(model, recurse=is_sequential_cpu_offload)
+                accelerate.hooks.remove_hook_from_module(model, recurse=True)
```

Comment:
Why always `recurse=True`? is this harmless regardless?

Reply:
I think it is fine to do that. We only remove hooks associated with accelerate.